lib/components/Dropdown/Dropdown.playroom.tsx

```diff
@@ -25,7 +25,6 @@ export const Dropdown = ({
   return (
     <BraidDropdown
       id={id ?? fallbackId}
-      name={name}
```

Comment:
This was breaking the build, not sure how it hasn't been a problem earlier. Same as `PasswordField` change below